db/migrate/20221227061816_remove_column_to_post.rb

```diff
@@ -0,0 +1,10 @@
+class RemoveColumnToPost < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :posts, :oc, :boolean
+    remove_column :posts, :spoiler, :boolean
+    remove_column :posts, :nsfw, :boolean
+    remove_column :posts, :oc_tag, :boolean
+    remove_column :posts, :spoiler_tag, :boolean
+    remove_column :posts, :nsfw_tag, :boolean
+  end
+end
```

Comment:
why do you have remove columns in a separate migration file? @SrinandhiniM

Reply:
I created those columns at first and found they are not needed so i removed in a separate file

Reply:
Can you delete this file if it is not required?